src/platform/intel/cavs/include/cavs/lib/memory.h

```diff
@@ -65,7 +65,7 @@
 #define HEAP_BUF_ALIGNMENT		PLATFORM_DCACHE_ALIGN
 
 /** \brief EDF task's default stack size in bytes. */
-#define PLATFORM_TASK_DEFAULT_STACK_SIZE	3072
+#define PLATFORM_TASK_DEFAULT_STACK_SIZE	0x1000
```

Comment:
Agree with this change. Would be real nice to have some actual data and a stack trace to demonstrate this.